base/services/tftpd/tftpd.h

```diff
@@ -30,138 +36,133 @@
 #endif
 
 //Constants
-#define my_inet_addr     inet_addr
+#define my_inet_addr 	inet_addr
 #define MAX_SERVERS 8
 
 //Structs
 struct home
 {
-    char alias[64];
-    char target[256];
+	char alias[64];
+	char target[256];
```

Comment:
This and onwards: feel free to look at our [Coding Style](https://reactos.org/wiki/Coding_Style). Do not `\t` for indentation, please convert this to spaces. The comment will not be repeated to avoid cluttering.